packages/test/local-server-stress-tests/src/test/localServerStressOrderSequentially.spec.ts

```diff
@@ -0,0 +1,97 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { done, isOperationType, takeAsync } from "@fluid-private/stochastic-test-utils";
+
+import { makeGenerator, reducer, saveFailures, type StressOperations } from "../baseModel.js";
+import {
+	convertToRealHandles,
+	covertLocalServerStateToDdsState,
+	DDSModelOpGenerator,
+	loadAllHandles,
+	validateConsistencyOfAllDDS,
+	type DDSModelOp,
+	type OrderSequentially,
+} from "../ddsOperations.js";
+import {
+	createLocalServerStressSuite,
+	LocalServerStressModel,
+	type LocalServerStressState,
+} from "../localServerStressHarness.js";
+
+const orderSequentiallyReducer = async (
+	state: LocalServerStressState,
+	op: OrderSequentially,
+) => {
+	const { baseModel, taggedHandles } = await loadAllHandles(state);
```

Comment:
I'm confused about the concept here - The `state` has a `channel` - which channel is it?  It appears to be one of the DDSes (a random one?), and we get the model for that DDS along with all the handles anywhere.

Then we take the operations from the OrderSequentially operation, realize the handles, and pass it to the reducer.  How is it that those operations are valid for that DDS?  I haven't looked at how the OrderSequentially operation is created, maybe that sheds some light.

Can you help explain?

Reply:
the client selection mixin in the harnes picks a client, datatore, and channel, and puts them on the state for all operations to use. it will be random.

the generator and reducer will always get the same state. the generator picks thing, and the reducer executes things. so the generator picks operations for the selected channel, and then the reducer runs those operations on the channel within the context of order sequentially.

Reply:
Ok cool, the client selection mixin is what I was missing, thanks.